google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQuery.java

```diff
@@ -939,74 +994,80 @@ public static QueryResultsOption maxWaitTime(long maxWaitTime) {
    *
    * <p>Example of running a query.
    * <pre> {@code
-   * String query = "SELECT unique(corpus) FROM [bigquery-public-data:samples.shakespeare]";
-   * QueryRequest request = QueryRequest.of(query);
-   * QueryResponse response = bigquery.query(request);
-   * // Wait for things to finish
-   * while (!response.jobCompleted()) {
-   *   Thread.sleep(1000);
-   *   response = bigquery.getQueryResults(response.getJobId());
-   * }
+   * String query = "SELECT distinct(corpus) FROM `bigquery-public-data.samples.shakespeare`";
+   * QueryJobConfiguration queryConfig = QueryJobConfiguration.of(query);
+   *
+   * // To run the legacy syntax queries use the following code instead:
```

Comment:
Also doesn't the query have slightly different syntax too?

Reply:
Yes, updated the comment with corresponding legacy syntax.